src/util/parseArgvAndInvoke.js

```diff
@@ -0,0 +1,32 @@
+import assertFunctions from './assertFunctions'
+import defaultInvokeOptions from './defaultInvokeOptions'
+
+export default function parseArgvAndInvoke(parse, usage, invokeFn) {
```

Comment:
Love this pattern, but I'm not wild about this name. This function doesn't "parse argv and invoke" it creates and returns a function that does that. Maybe `defineCommandInvoker` or `buildInvoker` or something.